src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Rectangle extends FigCol {
+    private double width;
+    private double height;
+
+    public Rectangle(double width, double height, Color color) {
+        super(color);
+        this.height = height;
+        this.width = width;
+    }
+
+    public double getWidth() {
+        return width;
+    }
+
+    public double getHeight() {
+        return height;
+    }
+
+    @Override
+    public double getArea() {
+        return width * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: " + getArea()
+                + " sq. units, width: " + width + " units, height: "
+                + height + " units, color: " + getColor());
```

Comment:
Use the 'name()' method instead of 'toString()' for getting String representation of enum constants. Replace 'getColor()' with 'getColor().name()' if 'getColor()' returns an enum type.